### PR TITLE
chore: bump web to v8.0.0-alpha.8

### PR DESCRIPTION
**Pull request by JammingBen**
Includes the last missing items for the current sprint (see down below) as well as the completed CERN merge.

* Enhancement [owncloud/web#9890](https://github.com/owncloud/web/pull/9890) - Create shortcuts
* Enhancement [owncloud/web#9905](https://github.com/owncloud/web/pull/9905) - Manage tags in details panel
* Enhancement [owncloud/web#9912](https://github.com/owncloud/web/pull/9912) - Add media type filter chip

Note that we updated pnpm to `v8.10.5` in Web. To avoid bumping the pnpm version in our nodejs docker image every time we do this, the pipeline now determines the used pnpm version and installs it before running `pnpm install`.

### CODE REVIEW

File: .drone.env [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # The test runner source for UI tests
-WEB_COMMITID=67ced24b3a6f06b767ff8111537b4a0b85c40993
-WEB_BRANCH=changeRespCode
+WEB_COMMITID=6514050264308fc630abf0fbf5b4dcf728ab4f44
+WEB_BRANCH=master
```
Comment (JammingBen) [inline .drone.env:3]: @2403905 @ScharfViktor I suppose this goes back to master eventually?

Comment (ScharfViktor) [reply .drone.env:3]: Yes, after merging this one https://github.com/owncloud/web/pull/9934

Comment (saw-jan) [reply .drone.env:3]: PR merged

**Review by fschade — APPROVED** [review]
LGTM

**Pull request merged**